docs/getting-started/prerequisites.md

```diff
@@ -0,0 +1,237 @@
+# Setting up the prerequisites
+
+This guideline includes the steps to install the prerequisites with specified versions for the Carbyne Stack. All installation procedures are done with the Terminal application of Ubuntu. **Please open the Terminal application before you proceed.**
+
+## Prerequisites
+
+This part of the tutorial is developed and tested with Ubuntu 20.04. Please refer to this [link](https://ubuntu.com/tutorials/install-ubuntu-desktop) for Ubuntu installation steps.
+
+## Platform Setup Prerequisites
+
+Software to be installed:
+
+- [go](https://go.dev/doc/install)
+- [Docker Engine](https://docs.docker.com/engine/install/ubuntu/) v20.10.6
+- [Kind](https://kind.sigs.k8s.io/) v0.11.0
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) v1.21.1
+- [Helm](https://helm.sh/docs/intro/install/) v3.7.1
+
+### go
+
+The go language is a prerequisite for the Docker Engine. In this guideline go 1.18 will be installed.
+
+1. Download go for Linux.
+
+    ```shell
+    wget https://go.dev/dl/go1.18.linux-amd64.tar.gz
+    ```
+
+2. Extract to /usr/local. You may need to insert the password for sudo permissions.
+
+    ```shell
+    sudo tar -C /usr/local -xzf go1.18.linux-amd64.tar.gz
+    ```
+
+3. Check if there is a “go” folder in "/usr/local".
+
+    ```shell
+    ls /usr/local
+    ```
+ 
+4. Update PATH for go.
+
+    ```shell
+    echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc
+    source ~/.bashrc
+    ```
+
+5. Verify if PATH is updated. Now go 1.18 is successfully installed.
+
+    ```shell
+    go version
+    ```
+
+### Docker Engine
+
+1. Update repository index.
```

Comment:
Suggestion: `Update repository index and install dependencies.`

Reply:
I have changed it to the suggested line.